docker/cea.config

```diff
@@ -0,0 +1,288 @@
+[general]
```

Comment:
Wait. Why do we need to provide the config file?

Reply:
to be able to pre-set some stuff. like the default project folder etc. the one normally shipped is geared towards windows.